janitor/functions/filter.py

```diff
@@ -33,29 +34,50 @@ def filter_string(
 
         >>> import pandas as pd
         >>> import janitor
-        >>> df = pd.DataFrame({"a": range(3, 6), "b": ["bear", "peel", "sail"]})
+        >>> df = pd.DataFrame({"a": range(3, 6), "b": ["bear", "peeL", "sail"]})
         >>> df
            a     b
         0  3  bear
-        1  4  peel
+        1  4  peeL
         2  5  sail
         >>> df.filter_string(column_name="b", search_string="ee")
            a     b
-        1  4  peel
+        1  4  peeL
+        >>> df.filter_string(column_name="b", search_string="L", case=False)
+           a     b
+        1  4  peeL
+        2  5  sail
+
+    Example: Filter names does not contain `'.'` (disable regex mode).
+
+        >>> import pandas as pd
+        >>> import janitor
+        >>> df = pd.Series(["JoseChen", "Brian.Salvi"], name="Name").to_frame()
+        >>> df
+                  Name
+        0     JoseChen
+        1  Brian.Salvi
+        >>> df.filter_string(column_name="Name", search_string=".", regex=False, complement=True)
+               Name
+        0  JoseChen
 
     `search_string` is also permitted to be any valid regex pattern.
 
     :param df: A pandas DataFrame.
     :param column_name: The column to filter. The column should contain strings.
-    :param search_string: A regex pattern or a (sub-)string to search.
+    :param search_string: A regex pattern or a (sub-)string to search. It's also
+        `str.contains` parameter `pat`'s alias name.
     :param complement: Whether to return the complement of the filter or not. If
         set to True, then the rows for which the string search fails are retained
         instead.
+    :param kwargs: `str.contains`'s keyword arguements.
```

Comment:
```suggestion
    :param kwargs: `str.contains`'s keyword arguments.
```

In the case we decide to use kwargs, please fix the spelling of "arguments".